doc/en/announce/release-2.5.0.rst

```diff
@@ -83,7 +83,7 @@ holger krekel
   Thanks Ralph Schmitt for the precise failure example.
 
 - fix issue244 by implementing special index for parameters to only use
-  indices for paramentrized test ids
+  indices for parametrized test ids
```

Comment:
there's an amusing thing elsewhere about a serious of variations of this word, but this isn't one of them...